Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,29 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if sys.platform == "linux":
+        return True
```

Comment:
Do you know if it is a "recent" Linux feature? Should we check for the Linux kernel version?

If you don't know, just leave it unchanged. We can fix it later, if needed.

Reply:
They were introduced in Linux 2.2. Do you think we should do a version check?

Reply:
Linux 2.2 is now obsolete, I don't think that anyone will attempt to run Python 3.9 on Linux 2.2. If they do, I expect way more problems than just abstract sockets :-)